crates/containerd-shim-wasmtime/Cargo.toml

```diff
@@ -6,10 +6,11 @@ edition.workspace = true
 [dependencies]
 anyhow = { workspace = true }
 containerd-shim-wasm = { workspace = true, features = ["opentelemetry"] }
+libc = { workspace = true }
 log = { workspace = true }
 hyper = { workspace = true }
 tokio = { workspace = true, features = ["signal", "macros"] }
-libc = { workspace = true }
+tokio-util = { workspace = true, features = ["rt"] }
```

Comment:
I was reading the difference between `JoinSet` and `TaskTracker` but I didn't understand if our use cases require some of the unique properties that `TaskTracker` brings. If not, I'd suggest we go with `JoinSet` since it meant one less dependency for runwasi.

Reply:
The property we leverage is

> 4. A TaskTracker can be cloned to share it with many tasks.

The tracker is cloned and passed to handler to track the tasks spawned there as well:
https://github.com/containerd/runwasi/pull/705/files#diff-b2d23b367806b354ef48f11bbfd796edae1c54af02e2a28f83fbbdf273b955b6R70

I think `JoinSet` is not `Clone` but I will double check.